app/controllers/api/admin/green_lanes/exempting_certificate_overrides_controller.rb

```diff
@@ -0,0 +1,80 @@
+module Api
+  module Admin
+    module GreenLanes
+      class ExemptingCertificateOverridesController < AdminController
+        include Pageable
+        include XiOnly
+
+        before_action :check_service, :authenticate_user!
+
+        def index
+          render json: serialize(exempting_certificate_override.to_a, pagination_meta)
+        end
+
+        def show
+          eco = ::GreenLanes::ExemptingCertificateOverride.with_pk!(params[:id])
+          render json: serialize(eco)
+        end
+
+        def create
+          eco = ::GreenLanes::ExemptingCertificateOverride.new(eco_params)
+
+          if eco.valid? && eco.save
+            render json: serialize(eco),
+                   location: api_admin_green_lanes_exempting_certificate_override_url(eco.id),
+                   status: :created
+          else
+            render json: serialize_errors(eco),
+                   status: :unprocessable_entity
+          end
+        end
+
+        def update
```

Comment:
After refinement I don't think we need to support the `#update` so this action can be dropped

Reply:
Done